### DISCUSSION TITLE
How come I have so many commits?

### DISCUSSION THREAD

**Discussion by eggbean** [anuraghazra/github-readme-stats#3903] [Q&A] [answered]
Could anyone explain why my stats say that I have made 2,684 commits in the last year? That seems like a lot more than I can see if I look at my main repositories. I guess that rebases are included and I do a lot of rebasing, but it still seems like much more than I expect.

**Comment by iibrahemali** [comment, marked as answer]
yeah, that commit count can add up fast for a few reasons. total culprit—it’s basically making new commits every time. also, if you’re working across multiple repos, private ones, or forks, those commits count too. plus, if you’ve got multiple emails linked to your git, that all gets thrown into the mix. and any automated stuff, like bots, will bump up the numbers as well. So yeah, that’s prob why it seems higher than you expected. and you can actually check the detailed commits on any day you want.
